code/server/routes/api.js

```diff
@@ -43,6 +44,24 @@ router.get('/randomUser', function(req, res){
 	});
 });
 
+// Getting a specific user
+// Specify username in GET variable
+router.get('/getUser', function(req, res) {
+
+    if (req.query.user) {
+        User.getUser(req.query.user).then(function(user) {
+            if (user) {
+              delete user.dataValues.password // probably not the best idea to send this over the wire
```

Comment:
Yeah, it should be there. I'll add it in. I fucking hate JS. It needs to decide if it wants the semicolon or not...